src/dropdown/dropdown.js

```diff
@@ -187,17 +187,30 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
     }
   };
 
+  function toggleDropdownUp() {
```

Comment:
Because outerHieght and outerWidth are jQuery methods and we only support jQLite or native element methods and properties.  Besides, the outerHeight/width jQuery documentation states:

Get the current computed height for the first element in the set of matched elements, including padding, border, and optionally margin.

And the HtmlElement offsetHeight/width documentation from MDN states:

Typically, an element's offsetHeight is a measurement which includes the element borders, the element vertical padding, the element horizontal scrollbar (if present, if rendered) and the element CSS height.

So the 2 are the same unless you include the margin parameter.